src/main/java/com/candyshop/islodycze/registration/UserRepository.java

```diff
@@ -2,6 +2,10 @@
 
 import com.candyshop.islodycze.model.User;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface UserRepository extends JpaRepository<User, Long> {
+
+    @Query("SELECT u FROM User u WHERE u.email = ?1")
+    User findByEmail(String email);
```

Comment:
Did you need to use Query? Wouldn't custom data JPA function like:
  List<User> findByEmail(String email);
 be enought?

Reply:
Right, i removed it